pkg/sql/logictest/testdata/logic_test/tenant_builtins

```diff
@@ -4,7 +4,7 @@ query IBIT colnames
 SELECT id, active, length(info), name FROM system.tenants ORDER BY id
 ----
 id  active  length  name
-1   true    10      system
+1   true    14      system
```

Comment:
Why did adding the fields to the proto -- but not setting them -- make this go up?

Reply:
My 80% sure guess is that this is the result of gogoproto.nullable = false.  While the new fields are "optional", by default we'll be putting zero-valued messages in both the new hlc.Timestamp and the enclosing message.  Both of those likely have 1 byte for the tag and 1 byte for the lengths. 2*(1+1) = 4.

Reply:
I _thought_ proto didn't encode fields that had their default/zero value?

> Also note that if a scalar message field is set to its default, the value will not be serialized on the wire.

Reply:
We could remove nullable=false from the top level message and probably get these bytes back in the default case.  We'll have to do a nil-check before accessing that field, but not on any of the fields inside of it which seems like a fine tradeoff.

Reply:
maybe we should just make the message type nullable; I'm not going to lose sleep over one alloc per tenant?

Reply:
> I thought proto didn't encode fields that had their default/zero value?

I have a feeling this is the result of proto2's optional and gogoproto's nullable kinda being at cross purpose.

Reply:
Nullable it is.